src/support/file.h

```diff
@@ -29,15 +29,15 @@
 
 namespace wasm {
 template <typename T>
-T read_file(const std::string &filename, bool debug);
+T read_file(const std::string &filename, bool binary, bool debug);
```

Comment:
It's more about expressing intent: what does binary=false mean? Not a big issue, I just find it easier to document it that way.